README.md

```diff
@@ -232,6 +244,31 @@ repo in this order:
    (assuming `origin` is the `knative.dev` repo)
 1. Remove the git branch (if any) from the Github UI
 
+### Post-release work
+
+#### Client Homebrew repo
+
+After the client release, the [Homebrew tap](https://github.com/knative/homebrew-client) needs to be updated with the new release:
+
+* Copy `kn.rb` to the `kn@${PREV_RELEASE}.rb` with `$PREV_RELEASE` to be replace with the latest release (e.g. `0.19`).
```

Comment:
I wonder if in the future, we can look at productivity to automate these keyed off the release variables there?

Reply:
Yep, that's on the roadmap https://github.com/knative/homebrew-client/issues/18 

Shouldn't be hard to write at least a script for that, just no time ;-(